pkg/csi/service/vanilla/controller.go

```diff
@@ -1244,11 +1244,16 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 		}
 		// Block Volume.
 		volumeType = prometheus.PrometheusBlockVolumeType
-		var node *cnsvsphere.VirtualMachine
+		var nodevm *cnsvsphere.VirtualMachine
 		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.UseCSINodeId) {
-			node, err = c.nodeMgr.GetNodeByUuid(ctx, req.NodeId)
+			// if node is not yet updated to run the release of the driver publishing Node VM UUID as Node ID
+			// look up Node by name
+			nodevm, err = c.nodeMgr.GetNodeByName(ctx, req.NodeId)
+			if err == node.ErrNodeNotFound {
+				nodevm, err = c.nodeMgr.GetNodeByUuid(ctx, req.NodeId)
```

Comment:
Can we add an Info message here to say we are falling back to Node UUID for lookup?

Reply:
done. added following message

```
log.Infof("Performing node VM lookup using node VM UUID: %q", req.NodeId)
```